errbot/core_plugins/flows.plug

```diff
@@ -0,0 +1,10 @@
+[Core]
```

Comment:
It is a little strange to have [Core] section, and Core flag equals 'true' (please excuse my ignorance).